### ISSUE TITLE
Can you meet with the Linux Foundation OpenSSF Digital Identity Attestation Working Group?

### ISSUE THREAD

**Issue by david-a-wheeler** [JanssenProject/home#2]
Hi, welcome to the Linux Foundation!

Would some folks from the Janssen project be willing to present & talk with the [Open Source Security Foundation (OpenSSF) Digital Identity Attestation Working Group (WG)](https://github.com/ossf/wg-digital-identity-attestation)?

We just met December 9. We're scheduled for December 23 but I think that will be cancelled. Our next meeting (after that) is January 6, 2021, at noon Eastern Time. I think it'd be good to talk; I suspect there are many ways we could work together. At the least, we'd like to hear a summary about what you're doing.

You can see the [WG notes](https://docs.google.com/document/d/1xPs2sSbH3I9Ich7OyLOzl85oJshnK8Q6WoAgREE5-zA/edit?ts=5fd115f4#heading=h.12hbj4l1tzth).

**Comment by nynymike** [comment]
I'd be happy to join the call on Jan 6. To build trust, you need both "tools and rules".  Any authentication/authorization system (like Janssen auth-server) is only useful after identity proofing has taken place, and credentials are securely delivered to the right person. In this sense, Janssen is a tool, but maybe not a tool that will help you with your core challenge, which sounds like it is "identity proofing".  While Janssen auth-server might not help, I'm happy to share how we identity proof developers, and the range of solutions I've seen at some of the organizations I've worked with over the past 20 years or so.

**Comment by david-a-wheeler** [comment]
Excellent! The WG page has the details. You can also email me dwheeler AT linuxfoundation DOT org. We're looking forward to it. I'd suggest ~10-15 minute presentation + Q&A time, that should at least enable a discussion. We normally record our meetings so others can watch later, let us know if that's a problem.

**Comment by david-a-wheeler** [comment]
@nynymike - Mike, are you still planning to present to us Jan 6 (tomorrow)?

**Comment by nynymike** [comment]
Yes, I have it on my schedule for 11am US Central time on Google Hangouts.
